packages/ogmios/package.json

```
@@ -52,7 +52,7 @@
     "prepack": "yarn build"
   },
   "devDependencies": {
-    "@cardano-ogmios/schema": "5.6.0",
+    "@cardano-ogmios/schema": "^5.6.0",
```

We shouldn't depend on ^ range because we're using internal functions directly
```suggestion
    "@cardano-ogmios/schema": "5.6.0",
```